tests/unit/client_test.py

```
@@ -184,4 +186,47 @@ def test_from_env_with_malformed_operator_key():
     with patch.object(client_module, 'load_dotenv'):
         with patch.dict(os.environ, env_vars, clear=True):
             with pytest.raises(ValueError):
-                Client.from_env()
\ No newline at end of file
+                Client.from_env()
+
+@pytest.mark.parametrize(
+    'valid_amount,expected',
```

Ruff (0.14.10)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)